src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  health = 100;
+  constructor(name) {
```

Comment:
Description from task:
`All instances of the Animal class must have properties health and a name. Health is 100 by default.`
So you should have `health` in `constructor`. Health default value can be separated to constant.